ci/docker/cuda-10.0-cpp.dockerfile

```diff
@@ -72,7 +72,9 @@ RUN apt-get update -y -q && \
 # - libgtest-dev only provide sources
 # - libprotobuf-dev only provide sources
 # - thrift is too old
-ENV ARROW_DEPENDENCY_SOURCE=SYSTEM \
+ENV PATH=/usr/lib/ccache/:$PATH \
```

Comment:
Could you sort this list alphabetically?

Reply:
Sure